File: ruby/front_back.rb
def front_back(string)
	first=string[0]
	last=string[-1]
	string[-1]=first
	string[0]=last
	return string
end
puts front_back("code") == "eodc" ? "pass" : "fail"
puts front_back("a") == "a" ? "pass" : "fail"
puts front_back("ab") == "ba" ? "pass" : "fail"

File: php/front_back.php
<?php
	function front_back($string)
	{
		$first=$string[0];
		$last=$string[strlen($string)-1];
		$string[0]=$last;
		$string[strlen($string)-1]=$first;
		return $string;
	}
	echo front_back("code")=="eodc" ? "pass" : "fail"; echo "\n";
	echo front_back("a")=="a" ? "pass" : "fail"; echo "\n";
	echo front_back("ab")=="ba" ? "pass" : "fail"; 

?> 
